crates/query-engine/sql/src/sql/convert.rs

```diff
@@ -315,17 +315,26 @@ impl From {
             From::Unnest {
                 expression,
                 alias,
-                column,
+                columns,
             } => {
                 sql.append_syntax("UNNEST");
                 sql.append_syntax("(");
                 expression.to_sql(sql);
                 sql.append_syntax(")");
                 sql.append_syntax(" AS ");
                 alias.to_sql(sql);
-                sql.append_syntax("(");
-                column.to_sql(sql);
-                sql.append_syntax(")");
+                if !columns.is_empty() {
```

Comment:
This change confused me a lot until I realized it is in a `From` clause.

Mulling it over a bit I think it's an alright change. But I wonder if it's ever necessary to give a column signature to `FROM UNNEST(..)`. Maybe we could do without it entirely?

Reply:
Will have a look at it in the future. I don't remember the exact case right now, but I think we use it for variables or something where we need to specify the names.

Reply:
And sorry about this, I'm working on a similar ticket and kind of got some changes mixed together.